lib/src/bindings_player_ffi.dart

```diff
@@ -451,6 +477,36 @@ class FlutterSoLoudFfi {
   );
   late final _setLooping = _setLoopingPtr.asFunction<void Function(int, int)>();
 
+  /// Get sound loop point value.
+  ///
+  /// [handle]
+  /// Returns the time in seconds.
+  double getLoopPoint(int handle) {
```

Comment:
I'm confused. Shouldn't there be two looping points? The start and the end? Or is it only possible to construct a looping region that ends with the end of the whole sound?

Reply:
Ok, reading https://solhsa.com/soloud/attributes.html, and yeah, there's just one loop point. This seems weird to me but no reason to change this if the C++ part doesn't support regions.

Reply:
I thought there was an ending point too, but there isn't. I think there is a way to do that, I will look at it later on, meanwhile it's something that could be useful. It is mentioned also [here](https://github.com/jarikomppa/soloud/issues/15#issuecomment-375354894).

My wish to have an endpoint, was to gain something similar to `sprite sheets` used in games, where a sequence of sprites are all in the same image. Likewise, an audio could have a bunch of different sounds in the same audio file.